docs/api/v3.rst

```diff
@@ -0,0 +1,558 @@
+                "python",
+                "packaging"
+            ]
+        }
+
+    :statuscode 204: Edited sucessfully
+    :statuscode 400: Some field is invalid
+    :statuscode 401: Not valid permissions
+
+
+
+Builds
+~~~~~~
+
+Builds are created by Read the Docs whenever a ``Project`` has its documentation built.
+Frequently this happens automatically via a web hook but can be triggered manually.
+
+Builds can be viewed in the build screen for a project.
+For example, here is `Pip's build screen`_.
+
+.. _Pip's build screen: https://readthedocs.org/projects/pip/builds/
+
+
+.. TODO:
+
+   for filtering by ``commit`` we need the build listing --the other cases are useless
+
+
+Build detail
+++++++++++++
+
+.. http:get::  /api/v3/build/(int:id)/
+
+    Retrieve details of a single build.
+
+    .. sourcecode:: js
+
+        {
+            "id": 7367364,
+            "date": "2018-06-19T15:15:59.135894",
+            "length": 59,
+            "state": "finished",
+            "state_display": "Finished",
+            "success": true,
+            "error": null,
+            "commit": "6f808d743fd6f6907ad3e2e969c88a549e76db30",
+            "docs_url": "http://pip.pypa.io/en/latest/",
+            "builder": "build03",
+            "cold_storage": false,
+            "links": [
+                {
+                    "href": "/api/v3/build/7367364/commands/",
+                    "rel": "commands",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/",
+                    "rel": "project",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/version/latest/",
+                    "rel": "version",
+                    "type": "GET"
+                }
+        }
+
+
+    :>json integer id: The ID of the build
+    :>json string date: The ISO-8601 datetime of the build.
+    :>json integer length: The length of the build in seconds.
+    :>json string type: The type of the build (one of "html", "pdf", "epub")
+    :>json string state: The state of the build (one of "triggered", "building", "installing", "cloning", or "finished")
+    :>json string state_display: The state of the build to be shown to the user
+    :>json boolean success: Whether the build was successful
+    :>json string error: An error message if the build was unsuccessful
+    :>json string commit: A version control identifier for this build (eg. the commit hash)
+    :>json string docs_url: The canonical URL of the build docs
+    :>json string builder: The hostname server that built the docs
+    :>json array links: Array with HEATEOAS_ links to retrieve related information
+
+    :statuscode 200: Success
+    :statuscode 404: There is no ``Build`` with this ID
+
+
+Build triggering
+++++++++++++++++
+
+.. TODO:
+
+   This endpoint may be under Project section
+
+.. http:post::  /api/v3/project/(string:slug)/builds/
```

Comment:
or under `/projects/versions/builds`?

Reply:
Without the slug?

If you missed the slug by mistake, I think it's not necessary to add `/versions/` to the URL since "the project has builds" and each "build is tied to a version"

Reply:
Yeah, I missed the slug